charts/rbac-definitions/Chart.yaml

```
@@ -0,0 +1,21 @@
+apiVersion: v2
+name: rbac-definitions
+description: Deploy rbac-manager RBACDefinitions via Helm
+type: application
+version: 0.1.0
+# matches apiVersion: rbacmanager.reactiveops.io/v1beta1
+appVersion: "1beta1"
+keywords:
+  - rbac-manager
+home: https://rbac-manager.docs.fairwinds.com/
```

it might make sense to deeplink the actual docs of the CRD
```suggestion
home: https://rbac-manager.docs.fairwinds.com/rbacdefinitions/
```